odl/phantom/noise.py

```diff
@@ -25,17 +25,62 @@
 import numpy as np
 
 
-__all__ = ('white_noise',)
+__all__ = ('white_noise', 'poisson_noise')
 
 
 def white_noise(space):
     """Standard gaussian noise in space, pointwise N(0, 1).
```

Comment:
Can't we have at least a `sigma` or `stddev` parameter so users don't have to think about proper scaling all the time? Shifting is trivial and not so important as feature. Both would be a simple change by using `np.random.normal(loc=mean, scale=stddev, size=space.shape)`.